Gemfile

```diff
@@ -52,6 +51,8 @@ group :development do
   # Display performance information such as SQL time and flame graphs for each request in your browser.
   # Can be configured to work on production as well see: https://github.com/MiniProfiler/rack-mini-profiler/blob/master/README.md
   gem 'annotate', '~> 3.1'
+  # Preview email in the default browser instead of sending it: https://github.com/ryanb/letter_opener
+  gem "letter_opener", :group => :development
```

Comment:
Is this gem using to the mail system?

Reply:
Yes, but only to do test confirmation to registration in development.